packages/daf-w3c/src/action-handler.ts

```diff
@@ -109,54 +114,80 @@ export interface PresentationInput {
   tag?: string
   expirationDate?: string
   verifiableCredential: string[]
+  [x: string]: any
 }
 
 const transformCredentialInput = (input: CredentialInput): VerifiableCredentialPayload => {
```

Comment:
What happens if `credentialSubject` is an array? I don't think we should support this at the moment but the spec supports it, so we should handle that case with an error.